src-ghc/Pact/GasModel/Utils.hs

```diff
@@ -284,9 +277,11 @@ toPactBinding  m = "{ " <> allKeys <> " }"
     allKeys = T.intercalate ", " $ map bindingFormat (HM.toList m)
     bindingFormat (key, _) = escapeText key <> " := " <> key
 
+-- Only use in testing: Pact.GasModel.GasTests
+--
 toPactKeyset :: T.Text -> T.Text -> Maybe T.Text -> A.Value
 toPactKeyset ksName ksValue predicate =
-  A.object [ksName A..= A.object ["keys" A..= [ksValue], "pred" A..= pred']]
+  toJSON $ HM.fromList [(ksName, A.object ["keys" A..= [ksValue], "pred" A..= pred'])]
```

Comment:
Since HashMap's are unordered, do we know, or do we have a test, that toJSON will be deterministic here?